string name
float32 heading
